docs/source/x_ext.rst

```diff
@@ -1287,19 +1299,19 @@ Handshake dependencies
 In order to avoid system level deadlock both the |processor| and the |coprocessor| shall obey the following rules:
 
 * The ``valid`` signal of a transaction shall not be dependent on the corresponding ``ready`` signal.
-* Transactions related to an earlier part of the instruction flow shall not depend on transactions with the same ``hartid`` and ``id`` related to a later part of the instruction flow. The instruction flow is defined from earlier to later as follows:
+* The only allowed dependencies between interfaces for transactions with the same ``hartid`` and ``id``are:
 
-  * compressed transaction
-  * issue transaction
-  * register transaction
-  * commit transaction
+  * ``issue_req`` may depend on ``compressed_resp``
+  * ``register`` may depend on ``compressed_resp``, ``issue_resp``, and ``commit``
+  * ``commit`` may depend on ``issue_resp``
+  * ``result`` may depend on ``issue_resp``, ``register``, and ``commit``
 
   .. only:: MemoryIf
 
-    * memory (request/response) transaction
-    * memory result transaction
+    * ``mem_req`` may depend on ``issue_req``, ``register``, and ``commit``
+    * ``mem_resp`` may depend on ``mem_req``
```

Comment:
Remove

Reply:
why? it is guarded by the memory interface parameter like anything else memory interface related

Reply:
It is guarded, but it is wrong as well

Reply:
Remove the line '* ``mem_resp`` may depend on ``mem_req``'. Obviously a response and req can depend on each other for the channel; we do not specify that for other handshakes.

Reply:
ah I see. My intention was for mem_result to be allowed to depend on mem_req. I think if I change that it should be fine.

Reply:
```suggestion
    * ``mem_result`` may depend on ``mem_req``
```